dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/MetricsGlobalRegistry.java

```diff
@@ -38,12 +37,10 @@ public class MetricsGlobalRegistry {
      * 3. Dubbo's own CompositeMeterRegistry is used by default
      */
     public static CompositeMeterRegistry getCompositeRegistry(ApplicationModel applicationModel) {
-        Optional<MetricsConfig> configOptional = applicationModel.getApplicationConfigManager().getMetrics();
-        if (configOptional.isPresent() && configOptional.get().getUseGlobalRegistry() != null && configOptional.get().getUseGlobalRegistry()) {
-            return Metrics.globalRegistry;
-        } else {
-            return compositeRegistry;
-        }
+        return applicationModel.getApplicationConfigManager().getMetrics()
+            .filter(MetricsConfig::getUseGlobalRegistry)
```

Comment:
useGlobalRegistry may be null, this filter will trigger NPE

Reply:
already fixed